tests/unit/algorithms/classification/test_helper.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
```

Comment:
I think you need to add __init__ files for each folders to avoid the conflict in pytest while collecting tests. It raises errors when same file names are detected.

Reply:
Also, the license header is missing in all the new files

Reply:
@sovrasov done